tools/test-backend

```diff
@@ -52,6 +52,9 @@ if __name__ == "__main__":
                       action="store_true",
                       default=False,
                       help="Show detailed output")
+    parser.add_option('--no-generate-fixtures', action="store_false", default=True,
+                      dest="generate_fixtures",
+                      help="Reduce running time by not calling generate-fixtures")
```

Comment:
Can you expand the help to explain that this may cause spurious failures for some tests?